templates/layouts/identified.jade

```diff
@@ -103,11 +106,12 @@ block content
                         a(href='/search/modifiedBefore='+meta.modified.description.split(' ')[0]+'&modifiedAfter='+meta.modified.description.split(' ')[0]+'&',title='Find all records modified on this date') 
                             span.fa.fa-search
 
-                if meta.description && meta.description != ''
+                if (meta.description && meta.description != '') || meta.canUpdate 
```

Comment:
Why not `&& meta.canUpdate` ?

Reply:
This allows you to add a description even when there is not one initially.